package.json

```diff
@@ -100,6 +108,10 @@
       {
         "view": "rHelpPages",
         "contents": "R Help Pages"
+      },
+      {
+        "view": "rLiveShare",
+        "contents": "R LiveShare not active."
```

Comment:
Microsoft refers to [Live Share](https://visualstudio.microsoft.com/services/live-share/) with spaces as opposed to LiveShare. Should this be 'R Live Share not active.'?